tests/test_jobs.py

```diff
@@ -594,8 +594,8 @@ def test_recommend_matches_all_individuals(self):
         }
 
         recommendations_expected = {
-            self.john_smith.individual.mk: [self.jsmith.individual.mk, self.js_alt.individual.mk],
-            self.jsmith.individual.mk: [self.js_alt.individual.mk],
+            self.js_alt.individual.mk: [self.jsmith.individual.mk, self.john_smith.individual.mk],
```

Comment:
Why does this test require the change?

Reply:
The order of the recommendations changed...